subprojects/docs/src/docs/userguide/running-builds/introduction/task_basics.adoc

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[task_basics]]
+= Task Basics
+
+A task represents some *atomic piece of work* that a build performs such as compiling classes, creating a JAR, generating Javadoc, or publishing archives to a repository.
+
+image::gradle-basic-5.png[]
+
+You run a Gradle `build` task using the `gradle` command or by invoking the Gradle Wrapper (`./gradlew' or `gradle.bat`) in your project directory:
```

Comment:
The quotes around
```
`./gradlew' or `gradle.bat`
```
seem off. I think there is a single-quote instead of a backtick